README.md

```diff
@@ -31,8 +39,11 @@ The following command will run a standalone instance of postgres found in the [e
 make test
 ```
 
-### Providers
-- [Nomad](https://registry.terraform.io/providers/hashicorp/nomad/latest/docs)
+### Verifying setup
+You can verify that postgres is running by checking the connection. This can be done using the `consul` docker image to set up a proxy. Check out the [required software](#required-software) section.
+```shell script
+make proxy-to-postgres
+```
```

Comment:
add something about how to check if the connection is OK as well

Reply:
Actually, considering this is a hotfix, Im OK with creating a new issue and leaving as is in this one, if you want, your call!

Reply:
Because it's you, I'll fix it :fire: